web_programming/crawl_google_results.py

```diff
@@ -1,24 +1,39 @@
 import sys
-import webbrowser
-
 import requests
-from bs4 import BeautifulSoup
 from fake_useragent import UserAgent
 
-if __name__ == "__main__":
-    print("Googling.....")
-    url = "https://www.google.com/search?q=" + " ".join(sys.argv[1:])
-    res = requests.get(url, headers={"UserAgent": UserAgent().random})
-    # res.raise_for_status()
-    with open("project1a.html", "wb") as out_file:  # only for knowing the class
-        for data in res.iter_content(10000):
+
+def crawl_and_store_google_search_result(query="potato", filename=""):
+    """This function will crawl google search result for the given query and
+    store it in a file.
+
+    Args:
+        query (str, optional): The search term provided by the user. Defaults
```

Comment:
```suggestion
        query: The search term provided by the user. Defaults
```
Document data types with type hints as discussed in CONTRIBUTING.md.